auth-tokens/src/main/java/com/palantir/tokens2/auth/UnverifiedJsonWebToken.java

```diff
@@ -43,14 +41,7 @@
 public abstract class UnverifiedJsonWebToken {
```

Comment:
I am not sure what exactly do you have in mind for this class. My main concern is that it makes some assumptions on the format of the Bearer token. We can discuss this as part of a separate PR/issue.